src/core/preferences.cpp

```diff
@@ -208,6 +208,23 @@ void Preferences::setValue(const QString &key, const QVariant &value)
     m_data.insert(key, value);
 }
 
+// UI options
+int Preferences::getTransferListFontScale() const
+{
+    int fontScale = value("Preferences/UI/TransferListFontScale", 100).toInt();
+    if (fontScale < 100)
+        fontScale = 100;
+    else if (fontScale > 200)
+        fontScale = 200;
+
+    return fontScale;
+}
+
+void Preferences::setTransferListFontScale(int scale)
+{
+    setValue("Preferences/UI/TransferListFontScale", scale);
```

Comment:
Since I grouped it in the GUI with the other TransferList elements this should go under `"Preferences/General/TransferListFontScale"` IMO.
Plus, the functions should be moved together with the other ones.